app/src/main/res/values/strings.xml

```diff
@@ -179,6 +182,7 @@
     <string name="dialog_unfollow_warning">Unfollow this account?</string>
     <string name="dialog_delete_toot_warning">Delete this toot?</string>
     <string name="dialog_redraft_toot_warning">Delete and re-draft this toot?</string>
+    <string name="mute_domain_warning">Are you really, really sure you want to block all of %s? In most cases, a few targeted blocks or mutes are sufficient and preferable.</string>
```

Comment:
How about

Are you sure you want to block the entire <domain>? You will not see content from that domain in any public timelines or your notifications. Your followers from that domain will be removed.